internal/kafka/internal/presenters/kafka.go

```diff
@@ -31,28 +33,48 @@ func ConvertKafkaRequest(kafkaRequestPayload public.KafkaRequestPayload, dbKafka
 }
 
 // PresentKafkaRequest - create KafkaRequest in an appropriate format ready to be returned by the API
-func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest) public.KafkaRequest {
+func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest, config *config.KafkaConfig) public.KafkaRequest {
 	reference := PresentReference(kafkaRequest.ID, kafkaRequest)
-
+	var ingressThroughputPerSec, egressThroughputPerSec, maxDataRetentionPeriod string
+	var totalMaxConnections, maxPartitions, maxConnectionAttemptsPerSec int
+	if config != nil {
+		kafkaConfig, err := config.GetKafkaInstanceSize(kafkaRequest.InstanceType, kafkaRequest.SizeId)
+		if err != nil {
+			glog.Warning(err)
```

Comment:
If this happens, that means that in our database we have a plan (instanceType+SizeId) that doesn't exists. I believe this should be logged in sentry too. WDYT?

Reply:
@ziccardi Do you mean something like this? sentry.CaptureException(err)

Reply:
Yes, since I think it is something we should react to. @machi1990 @JameelB WDYT?

Reply:
> @ziccardi Do you mean something like this? sentry.CaptureException(err)

we could use our logger.log.Error() or logger.log.Warning() which logs the log to sentry too.

Reply:
See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/36e09f13dc46c4fbd97b54f233ca2a2b4c8e68a4/docs/error-handing.md#errors-should-be-logged-and-forward-to-sentry-in-central-places